com.unity.robotics.urdf-importer/Runtime/RuntimeImport/RuntimeURDF.cs

```diff
@@ -108,7 +108,15 @@ public static string AssetDatabase_CreateFolder(string parentFolder, string newF
 #if UNITY_EDITOR
         if (!IsRuntimeMode())
         {
-            return AssetDatabase.CreateFolder(parentFolder, newFolderName);
+            if (!AssetDatabase.IsValidFolder($"{parentFolder}/{newFolderName}"))
+            {
+                return AssetDatabase.CreateFolder(parentFolder, newFolderName);
+            }
+            else 
```

Comment:
Not 100% sure, but I think this logic may fail if the IsValidFolder() is false for reasons other than the parent folder not existing. e.g. if it is outside Assets folder, or access rights?

Reply:
i'm also not 100% sure! the [documentation](https://docs.unity3d.com/ScriptReference/AssetDatabase.IsValidFolder.html) says `Given a path to a folder, returns true if it exists, false otherwise.`, but with the caveat that `The given path is relative to the project folder.`--so it could still return false in the case it's provided an absolute path rather than the relative one. would you suggest a change to the debug log warning, or some other change?

Reply:
Yes, perhaps we can change the log just to say that it could create the directory, without emphasizing on the actual reason, if we are not 100% sure about the root cause.